config/intermatic/pe653.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product Revision="1" xmlns="https://github.com/OpenZWave/open-zwave">
+  <MetaData>
+    <MetaDataItem name="Name">PE653</MetaDataItem>
+    <MetaDataItem name="Description">MultiWave Five-Channel Wireless Remote Receiver</MetaDataItem>
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/0005:0653:5045</MetaDataItem>
+    <MetaDataItem name="ProductPic">images/intermatic/pe653.png</MetaDataItem>
+    <MetaDataItem id="0653" name="ZWProductPage" type="5045">https://products.z-wavealliance.org/products/89/</MetaDataItem>
+    <MetaDataItem id="0653" name="FrequencyName" type="5045">U.S. / Canada / Mexico</MetaDataItem>
+    <MetaDataItem id="0653" name="Identifier" type="5045">PE653</MetaDataItem>
+    <ChangeLog>
+      <Entry author="Jeff Brown - jeff.brown@gmail.com" date="29 June 2020" revision="1">Initial creation, tested with firmware v3.3</Entry>
+    </ChangeLog>
+  </MetaData>
+
+  <!-- This device must be polled.  It also has a strange habit of reporting
+       old state when queried immediately after setting a switch and then
+       reporting the correct state when queried a few moments later. -->
+
+  <!-- This device reports 6 switches but there are only 5 circuits.
+       The switch on the default endpoint is redundant with endpoint 1 so
+       we omit it from the list of instances. -->
+  <CommandClass id="32">
```

Comment:
This should be auto discovered. Its not needed.

Reply:
Per the comment, this is intentional.  When auto-discovered, the device presents 6 switch instances however physically there are only 5 of them.  By listing them explicitly I am able to suppress the redundant switch instance and label each of them in the same manner as they appear in the product documentation.

FWIW, when I first set up the device, I got fooled by the redundant switch instance and was really confused why the switches weren't controlling the correct loads.